### ISSUE TITLE
rust-docs

### ISSUE THREAD

**Issue by spamwax** [hobofan/rsdocs-dashing#1]
Is it possible to use this to actually create docset for Rust's own docs? in `~/.rustup/toolchains/stable-x86_64-apple-darwin/share/doc/rust`

**Comment by hobofan** [comment]
I haven't tried it, but maybe?

I personally haven't had the need for it, since all the docset browsers normally already have docsets for Rust available.

**Comment by spamwax** [comment]
Correct, but for some reason the newer versions of Rust doc that I download
through Dash is broken and does not show documentations for any "method"!
